tools/linux/blk/blk_init.mk

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+ifeq ($(strip $(LIBVMM)),)
+$(error LIBVMM must be specified)
+endif
```

Comment:
Why not do `LIBVMM ?= ../../`?

Reply:
https://github.com/au-ts/libvmm/pull/79#discussion_r1685920469

Reply:
You could do:
 ```
 LINUX_BLK_DIR := $(abs $(dir $(last ${MAKEFILES_LIST}))
 LIBVMM ?= $(realpath ${LINUX_BLK_DIR)}/../../)
 ```

Reply:
Done https://github.com/au-ts/libvmm/pull/79/commits/282cad92d36ff3fe73ecd9c747e7ae1a08ab7717